pydrad/parse/parse.py

```diff
@@ -525,6 +435,153 @@ def grid_edges_right(self) -> u.cm:
         """
         return self.grid_edges[1:]
 
+    @property
+    @u.quantity_input
+    def coordinate(self) -> u.cm:
+        """
+        Spatial coordinate, :math:`s`, in the field-aligned direction.
+
+        An alias for `~pydrad.parse.Profile.grid_centers`.
+        """
+        return self.grid_centers
+
+    @property
+    @u.quantity_input
+    def electron_mass_density(self) -> u.Unit('g cm-3'):
+        # TODO: Account for possible presence of both electron
+        # and ion mass densities. If the electron mass density
+        # is present in this file, it will shift all of the
+        # indices in the .amr file by one.
+        if 'electron_mass_density' in self._amr_data.colnames:
+            return self._amr_data['electron_mass_density']
+        return self.mass_density
+
+    @property
+    @u.quantity_input
+    def mass_density(self) -> u.Unit('g cm-3'):
+        r"""
+        Mass density, :math:`\rho`, as a function of :math:`s`.
+
+        .. note:: This is a conserved quantity in HYDRAD.
+        """
+        return self._amr_data['mass_density']
+
+    @property
+    @u.quantity_input
+    def momentum_density(self) -> u.Unit('g cm-2 s-1'):
+        r"""
+        Momentum density, :math:`\rho v`, as a function of :math:`s`.
+
+        .. note:: This is a conserved quantity in HYDRAD.
+        """
+        return self._amr_data['momentum_density']
+
+    @property
+    @u.quantity_input
+    def electron_energy_density(self) -> u.Unit('erg cm-3'):
+        r"""
+        Electron energy density, :math:`E_e`, as a function of :math:`s`.
```

Comment:
I did this initially, but removed it as I didn't want to imply that these quantities are derived from the pressure rather than being conserved quantities that come directly from the simulation. I did include this expression in the docstrings for the electron (and hydrogen) pressures since those quantities are derived from the respective energy densities using these expressions.